docs/source/ja/generation_strategies.md

```diff
@@ -168,29 +168,6 @@ An increasing sequence: one, two, three, four, five, six, seven, eight, nine, te
 ['I look forward to seeing you all again!\n\n\n\n\n\n\n\n\n\n\n']
 ```
 
-### Contrastive search
```

Comment:
Make sure you comment `build-doc` to check if the other languages build their docs well (see [this slack thread](https://huggingface.slack.com/archives/C01NE71C4F7/p1754312295647569))